backend/app/assets/javascripts/spree/backend/turbolinks-configuration.js.erb

```diff
@@ -0,0 +1,20 @@
+// comment line necessary for correct interpolation of use_turbolinks
+Turbolinks.supported = <%= Spree::Backend::Config.use_turbolinks %>;
```

Comment:
No problem, but I think that you might have updated `tvdeyen/turbolinks` instead of `solidusio/turbolinks` : )